Automattic/wp-calypso
# Sites Management Page: optimize thumbnails size

#### Proposed Changes

* Change the aspect ratio requested to mShots
* Request different sizes for row and tile images
* Use srcset and sizes to optimize images rendering
* Make SiteThumbnail `alt` prop required

#### Screenshot

**ROW**

| Before | After |
|---|---|
| ![Row Before](https://user-images.githubusercontent.com/779993/186676086-459a29c3-885f-4bea-a91c-6e6f3da65bb2.png)  |  ![Row After](https://user-images.githubusercontent.com/779993/186676259-4c64add1-f7ba-48ee-a689-c77d2c55ae41.png) |

Row Img HTML:

```html
<img class="site-thumbnail__image"
alt="Damp Cowfish"
sizes="108px"
src="https://s0.wp.com/mshots/v1/https%3A%2F%2Fdamp-cowfish.jurassic.ninja%3Fv%3D1661342400%26iframe%3Dtrue%26preview%3Dtrue%26hide_banners%3Dtrue?vpw=1200&amp;vph=1200&amp;w=108&amp;h=78&amp;countToRefresh=0"
srcset="https://s0.wp.com/mshots/v1/https%3A%2F%2Fdamp-cowfish.jurassic.ninja%3Fv%3D1661342400%26iframe%3Dtrue%26preview%3Dtrue%26hide_banners%3Dtrue?vpw=1200&amp;vph=1200&amp;w=108&amp;h=78&amp;countToRefresh=0 108w,
        https://s0.wp.com/mshots/v1/https%3A%2F%2Fdamp-cowfish.jurassic.ninja%3Fv%3D1661342400%26iframe%3Dtrue%26preview%3Dtrue%26hide_banners%3Dtrue?vpw=1200&amp;vph=1200&amp;w=216&amp;h=156&amp;countToRefresh=0 216w,
        https://s0.wp.com/mshots/v1/https%3A%2F%2Fdamp-cowfish.jurassic.ninja%3Fv%3D1661342400%26iframe%3Dtrue%26preview%3Dtrue%26hide_banners%3Dtrue?vpw=1200&amp;vph=1200&amp;w=324&amp;h=234&amp;countToRefresh=0 324w"
loading="lazy">
```

**TILE**

| Before | After |
|---|---|
| ![Tile Before](https://user-images.githubusercontent.com/779993/186677097-8438b005-95a2-40f3-b24c-2a3e85f12061.png)  |  ![Tile After](https://user-images.githubusercontent.com/779993/186677424-203eef4f-6483-492c-9331-fda6db344310.png) |

Tile Img HTML:

```html
<img class="site-thumbnail__image"
alt="Damp Cowfish"
sizes="(min-width: 1400px) 401px,
        (min-width: 960px) calc(33vw - 48px),
        (min-width: 660px) calc(50vw - 48px),
        calc(100vw - 32px)"
src="https://s0.wp.com/mshots/v1/https%3A%2F%2Fdamp-cowfish.jurassic.ninja%3Fv%3D1661342400%26iframe%3Dtrue%26preview%3Dtrue%26hide_banners%3Dtrue?vpw=1200&amp;vph=1200&amp;w=401&amp;h=275.6875&amp;countToRefresh=0"
srcset="https://s0.wp.com/mshots/v1/https%3A%2F%2Fdamp-cowfish.jurassic.ninja%3Fv%3D1661342400%26iframe%3Dtrue%26preview%3Dtrue%26hide_banners%3Dtrue?vpw=1200&amp;vph=1200&amp;w=401&amp;h=275.6875&amp;countToRefresh=0 401w,
        https://s0.wp.com/mshots/v1/https%3A%2F%2Fdamp-cowfish.jurassic.ninja%3Fv%3D1661342400%26iframe%3Dtrue%26preview%3Dtrue%26hide_banners%3Dtrue?vpw=1200&amp;vph=1200&amp;w=802&amp;h=551.375&amp;countToRefresh=0 802w,
        https://s0.wp.com/mshots/v1/https%3A%2F%2Fdamp-cowfish.jurassic.ninja%3Fv%3D1661342400%26iframe%3Dtrue%26preview%3Dtrue%26hide_banners%3Dtrue?vpw=1200&amp;vph=1200&amp;w=1203&amp;h=827.0625&amp;countToRefresh=0 1203w"
loading="lazy">
``` 

#### Testing Instructions



* Go to `/sites`
* Inspect the SiteThumbnail image
* Observe no extra pixels are requested 

#### Pre-merge Checklist






- Closes #66723
